logic/common/log_utils.py

```diff
@@ -0,0 +1,188 @@
+''' Utility functions for logging purposes
+
+    <To Import>
```

Comment:
I'm not a fan of the ankle brackets. Let's go with all CAPs like in the other cli's. I'm thinking something like this:

USAGE EXAMPLE:
import logic.common.utils_log as log
log.SetVerbosityLevel(1)
log.Info("hello world")